packages/components/src/theme/ComposerTheme/Components/Button.ts

```diff
@@ -46,7 +46,13 @@ const generateButton = (color: string, textColor?: string) => {
   };
 };
 
-const generateAltButton = (color: string) => {
+const generateAltButton = (color: string, size: string) => {
+  const altButtonStylesBySize = {
+    xs: { height: '1.5rem' },
+    s: { height: '2.25rem' },
+    m: { height: '2.5rem' },
+    l: { height: '3.25rem' },
```

Comment:
Why do we need this? The button should have the size based on the height of the inner content plus the padding

Reply:
yes, I agree, but in this case these variants have a border which increases the height, I tried using the `border-box` but it didn't work, this doesn't happen with the rest of the variants because they don't have border.

Reply:
Can't we solve this using a outline instead of a border? If we need multiple outlines we can use after/before pseudo elements

Reply:
Yes, I dont like this approach  either, @guilean  it already has that outline property and it doesnt work, I am gonna take a look at this again.

Reply:
Try the approach with the :after/:before pseudo element, that should allow us to have two outlines